libs/utils/languages.js

```diff
@@ -0,0 +1,22 @@
+export default {
```

Comment:
Will all of adobe.com share the same languageBased-map? If yes this belongs into utils.
If consumers will always set this themselves - move this into our scripts.

Adding extra requests for something that is required for LCP is a performance/code-maintenance tradeoff where performance wins.

Reply:
Hey Okan! 
Individual consumers will be having their own languageBased map. It's like the locales object, but for langauge based sites. 
I can put this into the scripts.js, however since the locales live in a separate file, I figured it made sense to do the same here thing. Thoughts?
Either way, this will only affect milo website performance and not consumer sites consuming milo.

Reply:
Consumers have everything together: https://github.com/adobecom/bacom/blob/stage/scripts/scripts.js
We should also move this, not for performance within milo itself - but consumers mostly just copy our setup.
We don't want something with performance penalties accidentally getting copied as is

Reply:
alrighty, I'll move it.